module/motion/QuinticWalk/src/QuinticWalk.cpp

```diff
@@ -124,7 +124,7 @@ namespace module::motion {
             }
         });
 
-        on<Configuration>("QuinticWalk.yaml").then([this](const Configuration& cfg) {
+        on<Configuration, MainThread>("QuinticWalk.yaml").then([this](const Configuration& cfg) {
```

Comment:
Adding `MainThread` to `on<Configuration>` should be a pointless and unneeded operation, pretty sure you don't need it for `on<Startup>` either